src/main/java/io/inversion/cloud/model/Column.java

```diff
@@ -64,6 +66,17 @@ public int compareTo(Column o)
       return 0;
    }
 
+   public boolean equals(Object object)
+   {
+      if (object instanceof Column)
+      {
+         Column column = (Column) object;
+         if ((table != null && table.equals(column.table)) && Utils.equal(name, column.name))
+            return true;
+      }
+      return object == this;
```

Comment:
Why not just return false?

Reply:
returning object==this was accurate but probably lazy...I'll change...same for the equals method in Index and Table